### PR TITLE
Shadowcodex website patch 3

### PR DESCRIPTION
**Pull request by shadowcodex**
Changes:

- Fixed Nav Shadow
- Fixed Spacing on footer for mobile and to not hide links from gitter sidecar
- Fixed section spacing to have a space after last section to maintain consistency
- Changed layout of console to fit within normal container and still break out the same on mobile
- All contents on index are within a single container now
- Fixed features list to be correctly nested
- Fixed Why Meteor Up to break out correctly on mobile
- Made the community statistics live stats pulled from github API
- Added CNAME for custom domain
- Added Google Analytics

### CODE REVIEW

File: docs/css/custom.css [hunk 1]
```diff
@@ -2,6 +2,10 @@ body {
     background-color: #141B2B;
 }
 
+.fix-shadow {
+    box-shadow: none;
+}
```
Comment (zodern) [inline docs/css/custom.css]: It looks better without the shadow. 

You can remove the shadow at  https://github.com/zodern/meteor-up/blob/f11970f9967b215b53216a81b3ffa7f2f0da7f1c/docs/css/custom.css#L156-L161

**Review by zodern — CHANGES_REQUESTED** [review]
Thank you. One small change before it can be merged.

File: docs/css/custom.css (hunk 1 above)
Comment (shadowcodex) [reply docs/css/custom.css]: Oh, I totally missed that.

**Comment by shadowcodex** [comment]
done.

**Review by zodern — APPROVED** [review]

**Pull request merged**